docs/data/elasticsearch/generated/transactions.json

```diff
@@ -63,6 +63,9 @@
                     "started": 43
                 },
                 "type": "request"
+            },
+            "user": {
+                "id": "123user"
```

Comment:
`user` information from the `metadata` is set if no `transaction.context.user` information was sent.